src/Umbraco.Core/HealthCheck/Checks/AbstractSettingsCheck.cs

```diff
@@ -0,0 +1,160 @@
+﻿using System;
```

Comment:
This is the same file as AbstractConfigCheck, but renamed (probably shouldn't have done that here)